src/clean.rs

```diff
@@ -172,32 +136,20 @@ mod tests {
             let result = Builder::new()
                 .input(Some("C:\\Cargo.toml"))
                 .build()
-                .target_wix();
-            assert!(result.is_err());
-        }
-
-        #[test]
-        fn target_wix_with_existing_file_but_not_cargo_toml_fails() {

```

Comment:
This test was removed because it was testing for an error scenario that's... kinda wrong. It is 100% allowed to have a Cargo.toml whose name isn't Cargo.toml, if you pass it with --manifest-path.